src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`3.cs

```diff
@@ -136,32 +136,20 @@ protected override void SetupNewTextView(IVsTextView textView)
                 {
                     // We need to get our outlining tag source to notify it to start blocking
                     var outliningTaggerProvider = this.Package.ComponentModel.GetService<OutliningTaggerProvider>();
-                    ProducerPopulatedTagSource<IOutliningRegionTag> tagSource;
-                    if (outliningTaggerProvider.TryRetrieveTagSource(wpfTextView, wpfTextView.TextBuffer, out tagSource))
+
+                    // If this file is a metadata-from-source file, we want to force-collapse
```

Comment:
@pharring, does this ultimately do what we want, perf-wise? I have remembered and forgotten from our conversation several times.